src/index.js

```diff
@@ -19,30 +19,48 @@ var setup = state.current();
 
 // get into dom asap to trigger asset loading
 function renderHolder() {
-    var url = util.resolveClientUrl(util.currentRequestId(), true);
-    var html = '<div class="glimpse"><a class="glimpse-icon" target="_blank" href="' + url + '"><div class="glimpse-icon-text">Glimpse</div></a><div class="glimpse-hud"></div></div>'
-    $(html).appendTo('body');
+    if (document.readyState === 'complete') {
```

Comment:
I'm a little concerned that we still need this check, given that it's called from `$(func)` which means [call this function when the DOM is ready](http://api.jquery.com/jquery/#jQuery3). Is this a bug in jQuery?